### PR TITLE
Remove pm2 from development workflow 

### PR DESCRIPTION
**Pull request by ender503**
The pm2 is a production managment tool.
We should focus on developing the TangleID so remove the pm2.
And it should be added in our production environment.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,6 @@
   },
   "scripts": {
     "preinstall": "npm install pm2 -g",
-    "heroku-postbuild": "npm run build",
```
Comment (jserv) [inline package.json]: It is independent from `pm2`, right?

Comment (ender503) [reply package.json]: Yes, this script is for Heroku platform, so it's useless for our deployment platform.

Comment (jserv) [reply package.json]: Decouple this commit from pull request. Let's focus on pm2-specific changes.

**Comment by ender503** [comment]
Thanks for your advice 👍

**Pull request merged**